La-Mula-App-main/src/main/kotlin/com/lamulaapp/controller/dto/RolesDto.kt

```diff
@@ -0,0 +1,14 @@
+package com.lamulaapp.controller.dto
+
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class RolesDto(

```

Comment:
Usar singular en las clases.

```suggestion
data class RoleDto(

```